extensions/webjars-locator/deployment/src/test/java/io/quarkus/webjar/locator/test/WebJarLocatorDevModeTest.java

```diff
@@ -50,7 +50,7 @@ public void testDevMode() {
                 .statusCode(200);
 
         // Test using version in url of existing Web Jar
-        RestAssured.get("/webjars/jquery/3.4.1/jquery.min.js").then()
+        RestAssured.get("/webjars/jquery/3.5.1/jquery.min.js").then()
```

Comment:
@gsmet Isn't that going to break as soon as someone bumps the value of the property?
A sustainable solution would be to reuse the property here.